src/resolvers/query/card/index.js

```diff
@@ -10,17 +10,17 @@ export default async function card(parent, args) {
   const response = await getStripeCard(stripeCustomerId);
```

Comment:
```suggestion
  const customerInfo = await getStripeCard(stripeCustomerId);
```

then you can delete the next line and just replace `response` below with `customerInfo`.